src/content/docs/cheerp/12-reference/40-browser-apis/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Browser APIs
+description: "`client.h`"
+# TODO: terrible page because we are explaining both client.h and clientlib.h
+---
+
+You can use browser APIs in [genericjs code](/cheerp/reference/sections/genericjs) via the `client` namespace.
+
+## `cheerp/clientlib.h`
+
+Browser APIs are provided by `clientlib.h`, which can be included with:
+
+```cpp
+#include <cheerp/clientlib.h>
+```
+
+## `cheerp/client.h`
+
+> [!warning] Cheerp nightly only
+> This header is currently only available on Cheerp nightly. Use `clientlib.h` on stable.
```

Comment:
this is not true. client.h was always there.
The difference with clientlib.h is that clientlib.h is JUST the generated browser APIs, while client.h also contains helpers like the SAFE_INLINE macro, the ``cheerp::*` helpers like `Callback` and makeArrayRef`, etc...